src/Service/DistanceMatrix/Request/DistanceMatrixRequest.php

```diff
@@ -504,7 +505,7 @@ public function setLanguage($language = null)
     public function buildQuery()
     {
         $locationBuilder = function (LocationInterface $location) {
-            return $location->buildQuery();
+            return $location->buildQuery() . ($location instanceof EncodedPolylineLocation ? ':' : '');
```

Comment:
@egeloen That's what I initially did but then tests for some other services failed so I came with this solution.